home/home_2.1/HelloWorld.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <unordered_map>
+#include <sstream>
+
+const std::string FILENAME = "user_stats.txt";
+
+std::unordered_map<std::string, int> readStats() {
+    std::unordered_map<std::string, int> stats;
+    std::ifstream infile(FILENAME);
+    std::string line;
+    while (std::getline(infile, line)) {
```

Comment:
it is better to encapsulate statistics into some structure or class. The saving and loading the data will be some methods and it will make a support of code easier. As we are using c++23 we can even do something like https://mobiarch.wordpress.com/2023/12/17/reading-a-file-line-by-line-using-c-ranges/ 

Just make design simple but easy extendable for example if someone asks to add more data to statistics you will need to have easy way to change the format and at the same time there may be a request to support old database, convert it to the new format and so on. Therefore with encapsulated logic it will be easier to do. Also later we will intensively use unit tests and as you understand - it is much more easy to test small isolated logic then a single big function with a lot of branches.